app/models/identity.rb

```diff
@@ -1,23 +1,57 @@
+# == Schema Information
+#
+# Table name: identities
+#
+#  id           :integer          not null, primary key
+#  provider     :string
+#  uid          :string
+#  person_id    :integer
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  access_token :string
+#
+
 class Identity < ActiveRecord::Base
   before_save :request_long_lived_token
 
   belongs_to :person
-  validates_presence_of :uid, :provider, :person_id
+  validates_presence_of :uid, :provider#, :person_id
```

Comment:
Did you mean to leave person_id commented out?

Reply:
yes, but i should be more explicit about why. (it's in a commit message somewhere.)

the reason is that you can't pass `Identity` as a nested attribute to `person` if it validates the presence of `person_id`. for it to be a nested attribute, you have to be able to construct it with `person_id` as nil, and then give it `person_id` once the person it is nested inside of has been saved.